src/renderer/css/components/Modal.scss

```diff
@@ -4,6 +4,20 @@
 
 @import '~@mattermost/compass-icons/css/compass-icons.css';
 
+::-webkit-scrollbar {
+    width: 6px;  // for vertical scrollbars
+    height: 6px; // for horizontal scrollbars
+}
+
+::-webkit-scrollbar-track {
+    background: rgba(0, 0, 0, 0);
+}
+
+::-webkit-scrollbar-thumb {
+    border-radius: var(--radius-s);
+    background: rgba(var(--center-channel-color-rgb), 0.4);
+}
+
```

Comment:
Firefox does not support `::-webkit-scrollbar`, `::-webkit-scrollbar-track`, `::-webkit-scrollbar-thumb`

https://caniuse.com/?search=%3A%3A-webkit-scrollbar